### PR TITLE
restructure starting scripts documentation, add vision/policy

### PR DESCRIPTION
**Pull request by mikapfl**
# Purpose of this PR

Rewrite `scripts/start/readme.txt` (now `scripts/start/README.md`):
* nice formatting as markdown like the tutorials
* remove language about the old starting scripts, no one remembers those anymore anyway :wink: 
* remove documentation on how to start REMIND, this is covered in the tutorials
* add a vision / policy how we want the starting scripts to look like (essentially, it should look more like we are used to from R packages)

What do others think?

## Type of change

- [x] This change is a documentation update

## Checklist:

- [ ] My code follows the coding etiquette
- [ ] I have performed a self-review of my own code
- [ ] Changes are commented, particularly in hard-to-understand areas
- [ ] I have updated the in-code documentation
- [ ] I have adjusted reporting where it was needed
- [ ] All automated model tests pass (`FAIL 0` in the output of `make test`)

### CODE REVIEW

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
> nice formatting as markdown like the tutorials

```
$ wc -L scripts/start/README.md
407 scripts/start/README.md
```
image
:p

**Review by dklein-pik — APPROVED** [review]

**Comment by mikapfl** [comment]
I made a tutorial out of it, as `07a_Advanced_ChangeStartingScripts.md`.

**Review by LaviniaBaumstark — APPROVED** [review]

File: tutorials/07a_Advanced_ChangeStartingScripts.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                  login node                           slurm job
+```
+
+### Detail
+
+```
+Rscript start.R
+  -> choose_slurmConfig()               [scripts/start/choose_slurmConfig.R]
+     configure_cfg(cfg, scenario, ...)  [start.R]
+     save cfg to runtitlte.RData (in REMIND's mainfolder)
```
Comment (orichters) [inline tutorials/07a_Advanced_ChangeStartingScripts.md:32]: should be runtitle I guess

File: tutorials/07a_Advanced_ChangeStartingScripts.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+     submit(cfg)                        [scripts/start/submit.R]
+       - create output folder
+       - copy scripts/start/prepare_and_run.R into results folder
+       - save cfg to config.RData into results folder
```
Comment (orichters) [inline tutorials/07a_Advanced_ChangeStartingScripts.md:36]: it's actually `config.Rdata` with a small `d`, as opposed to `runtitle.RData` which actually uses a capital `D`. Why ever.

**Review by orichters — APPROVED** [review]
Thanks!

File: tutorials/07a_Advanced_ChangeStartingScripts.md (hunk 2 above)
Comment (mikapfl) [reply tutorials/07a_Advanced_ChangeStartingScripts.md]: Thanks, your attention to detail is appreciated!

**Pull request merged**